info/v2/container.go

```diff
@@ -204,3 +207,27 @@ type NetworkStats struct {
 	// Network stats by interface.
 	Interfaces []v1.InterfaceStats `json:"interfaces,omitempty"`
 }
+
+// Instantaneous CPU stats
+type CpuInstStats struct {
+	Usage CpuInstUsage `json:"usage"`
+}
+
+// CPU usage time statistics.
+type CpuInstUsage struct {
+	// Total CPU usage.
+	// Units: nanocores per second
```

Comment:
I used nanocores/s for both precision and consistency with the other metrics, which are also nano.

Kubelet will expose milli, so yes, it will divide the numbers. Although it's not converting "back", since I obtain the instantaneous metrics from accumulated nanoseconds, not milliseconds.